confdgnmi/src/confd_gnmi_api_adapter.py

```diff
@@ -1,11 +1,15 @@
+import os
+import select
```

Comment:
Did you have a look at higher-level `selector` module instead? The same holds for `os` - you use it only for pipes, there is higher-level `multiprocessing.Pipe()`.

Reply:
No.